lib/ssf/local_buffering_client.rb

```diff
@@ -0,0 +1,14 @@
+module SSF
+  class LocalBufferingClient < Client
+
+    attr_reader :buffer
+
+    def initialize()
+      @buffer = []
+    end
+
+    def send_to_socket(message)
+      @buffer << message
```

Comment:
+1 on a `clear` method; at the very least it makes it clear that a new client should be initialized for every fixture in P-S